sample-install/portperbroker_plain/base/kroxylicious-config.yaml

```diff
@@ -23,7 +23,7 @@ data:
           config:
             bootstrapAddress: localhost:9292
             brokerAddressPattern: kroxylicious-service:$(portNumber)
-        logNetwork: false
+        logNetwork: true
```

Comment:
Do we want this set to `true` in the exemplars?

Reply:
Fixed.